### ISSUE TITLE
Use the console to print the event in vue2, and an error will be reported in the console

### ISSUE THREAD

**Issue by Gxc2xn** [wallabyjs/console-ninja#172]
### What happened?

In vue2, the event is printed in the callback function of the input event of the input box, and the console will display

VM977:1 [Vue warn]: Property or method "length" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property.

### Version

v0.0.182

### Steps to reproduce (or sample repo)

In vue2, the event is printed in the callback function of the input event of the input box, and the console will display

VM977:1 [Vue warn]: Property or method "length" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property.

### Log output

```shell
15:13:58.510 info buildHook-20507 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, live-server, nuxt, remix, qwik, hydrogen, serverless, astro, node
15:13:58.510 info buildHook-20507 running tools: webpack
15:13:58.511 info buildHook-20507 installing fs interceptor
15:13:59.080 info buildHook-20507 build hook already installed
15:13:59.544 info buildHook-20507 file processed (stat): 0 log points, error handler: true
15:13:59.550 info buildHook-20507 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, live-server, nuxt, remix, qwik, hydrogen, serverless, astro, node
15:13:59.550 info buildHook-20507 running tools: webpack
15:14:01.144 info buildHook-20507 file processed (stat): 4 log points, error handler: false
15:14:01.175 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:01.214 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:01.240 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:01.258 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:01.275 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:01.291 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:01.312 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:01.322 info buildHook-20507 file processed (stat): 2 log points, error handler: false
15:14:01.335 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:01.348 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:01.361 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:01.389 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:01.417 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:01.441 info buildHook-20507 file processed (stat): 9 log points, error handler: false
15:14:01.467 info buildHook-20507 file processed (stat): 10 log points, error handler: false
15:14:01.495 info buildHook-20507 file processed (stat): 10 log points, error handler: false
15:14:01.513 info buildHook-20507 file processed (stat): 4 log points, error handler: false
15:14:01.544 info buildHook-20507 file processed (stat): 2 log points, error handler: false
15:14:01.552 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:01.574 info buildHook-20507 file processed (stat): 13 log points, error handler: false
15:14:01.593 info buildHook-20507 file processed (stat): 12 log points, error handler: false
15:14:01.613 info buildHook-20507 file processed (stat): 9 log points, error handler: false
15:14:01.633 info buildHook-20507 file processed (stat): 10 log points, error handler: false
15:14:01.647 info buildHook-20507 file processed (stat): 9 log points, error handler: false
15:14:01.672 info buildHook-20507 file processed (stat): 16 log points, error handler: false
15:14:02.335 info buildHook-20507 file processed (stat): 0 log points, error handler: false
15:14:03.008 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:04.567 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:04.651 info buildHook-20507 file processed (stat): 6 log points, error handler: false
15:14:04.666 info buildHook-20507 file processed (stat): 11 log points, error handler: false
15:14:04.679 info buildHook-20507 file processed (stat): 7 log points, error handler: false
15:14:04.852 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:04.900 info buildHook-20507 file processed (stat): 2 log points, error handler: false
15:14:06.220 info buildHook-20507 file processed (stat): 2 log points, error handler: false
15:14:06.245 info buildHook-20507 file processed (stat): 2 log points, error handler: false
15:14:06.275 info buildHook-20507 file processed (stat): 8 log points, error handler: false
15:14:06.387 info buildHook-20507 file processed (stat): 6 log points, error handler: false
15:14:06.412 info buildHook-20507 file processed (stat): 3 log points, error handler: false
15:14:06.508 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:06.528 info buildHook-20507 file processed (stat): 2 log points, error handler: false
15:14:07.065 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:07.071 info buildHook-20507 file processed (stat): 3 log points, error handler: false
15:14:07.104 info buildHook-20507 file processed (stat): 3 log points, error handler: false
15:14:07.111 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:07.125 info buildHook-20507 file processed (stat): 0 log points, error handler: false
15:14:10.425 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:10.442 info buildHook-20507 file processed (stat): 1 log points, error handler: false
15:14:10.459 info buildHook-20507 file processed (stat): 3 log points, error handler: false
15:14:12.811 info host client connected: runtimeHook { address: '::', family: 'IPv6', port: 49775 }
15:14:13.559 info host client connected: runtimeHook { address: '::', family: 'IPv6', port: 49775 }
15:15:32.450 info host client disconnected: runtimeHook 1001
15:15:32.450 info host client disconnected: runtimeHook
15:15:32.450 info host client disconnected: runtimeHook 1001
15:15:32.451 info host client disconnected: runtimeHook
15:15:32.482 info host client connected: runtimeHook { address: '::', family: 'IPv6', port: 49775 }
15:15:33.186 info host client connected: runtimeHook { address: '::', family: 'IPv6', port: 49775 }
```

**Comment by smcenlly** [comment]
Unfortunately we cannot reproduce your issue. Could you please provide us with step-by-step instructions to reproduce your issue, or else a sample repo.

The steps we took were:

1. `npm install -g @vue/cli` (v5.0.8)
2. `vue create my-project`
3. Selected preset, `Default ([Vue 2] babel, eslint)`
4. Selected package manager, `NPM`
5. Open VS Code
6. In VS Code terminal, ran `npm run serve`
7. Changed `HelloWorld.vue` script tag to:

```vue
<script>
export default {
  name: 'HelloWorld',
  props: {
    msg: String
  }
}

const a = {
  b: 1
};

console.log(a);
</script>
```

8. Confirmed `console.log` is working.
9. Added Log Point (debug breakpoint) to `const a = {` and saw the value being output.

**Comment by Gxc2xn** [comment]
1.当我使用element-ui时https://element.eleme.cn/#/en-US
2.
<img width="283" alt="image" src="https://github.com/wallabyjs/console-ninja/assets/102852269/531d1ad4-e985-4e76-8c87-e4299a33c7f7">
3.
<img width="182" alt="image" src="https://github.com/wallabyjs/console-ninja/assets/102852269/1b83d223-37d2-4be0-ba51-e1ef5d3e80e7">

**Comment by smcenlly** [comment]
Thanks for the details. We've reproduced the issue. [Sample repo here](https://github.com/wallabyjs/cn-172) for reference.

**Comment by Gxc2xn** [comment]
这个会修复吗？
看了你的示例好像漏了关键的东西
![image](https://github.com/wallabyjs/console-ninja/assets/102852269/3c90bc78-888a-4022-8a67-42dde31178c0)

**Comment by smcenlly** [comment]
Thanks for spotting. We had already identified that was a problem.

**Comment by Gxc2xn** [comment]
希望能修复，谢谢！

**Comment by ArtemGovorov** [comment]
Thanks for reporting the issue. It's now fixed and the fix is published in v1.0.183+.
